config/oracle_jre.yml

```diff
@@ -20,7 +20,7 @@
 # e.g.  repository_root: "http://example.com/oracle-jre/{platform}/{architecture}"
 ---
 jre:
-  version: 1.8.0_+
+  version: 1.8.0_12+
```

Comment:
based on 'def micro_and_qualifiers(s)' in file 'tokenized_version.rb' I think the original 1.8.0_+ should work no matter how many digits the qualifier is. Having specified the repo to our own will give it a limited amount of options to choose from that we control.